src/main/resources/assets/carpet/lang/zh_cn.json

```diff
@@ -1,12 +1,208 @@
+  "rule.commandCameramode.extra.0": "/c和/s命令所有玩家（包括非OP玩家）都可以使用",
+  
+  "rule.commandDistance.name": "距离测量",
+  "rule.commandDistance.desc": "启用/distance命令以测量游戏中两点间距离",
+  "rule.commandDistance.extra.0": "可以通过放置棕色地毯的方式使用本功能，前提是“放置地毯执行功能”已启用",
+  
+  "rule.commandDraw.name": "画图",
+  "rule.commandDraw.desc": "启用/draw命令以允许绘制一些简单图形",
+  
+  "rule.commandInfo.name": "获取方块数据",
+  "rule.commandInfo.desc": "启用/info命令以获取方块数据",
+  "rule.commandInfo.extra.0": "可以通过放置灰色地毯的方式使用本功能，前提是“放置地毯执行功能”已启用",
+  
+  "rule.commandLog.name": "游戏数据监视器",
+  "rule.commandLog.desc": "启用/log命令以在聊天栏/Tab栏监视游戏部分数据",
+  
+  "rule.commandPerimeterInfo.name": "刷怪区域搜寻器",
+  "rule.commandPerimeterInfo.desc": "启用/perimeterinfo命令以搜寻周围的可刷怪区域",
+  
+  "rule.commandPlayer.name": "假人控制",
+  "rule.commandPlayer.desc": "启用/player命令以控制/召唤玩家",
+  
+  "rule.commandScript.name": "地毯脚本控制器",
+  "rule.commandScript.desc": "启用/script相关命令",
+  "rule.commandScript.extra.0": "一个Scarpet语言的游戏内API",
+  
+  "rule.commandSpawn.name": "刷怪追踪器",
+  "rule.commandSpawn.desc": "启用/spawn命令用于怪物生成追踪",
+  
+  "rule.commandTick.name": "游戏时钟控制器",
+  "rule.commandTick.desc": "启用/tick命令以控制游戏时钟",
+  
+  "rule.commandTrackAI.name": "AI追踪器",
+  "rule.commandTrackAI.desc": "启用/track命令以追踪生物的AI",
+  
+  "rule.ctrlQCraftingFix.name": "ctrlQ合成修复",
+  "rule.ctrlQCraftingFix.desc": "在合成时允许使用ctrl+Q快捷键扔出结果格内所有物品",
+  
+  "rule.customMOTD.name": "MOTD更改",
+  "rule.customMOTD.desc": "设置一个不同的MOTD信息",
+  "rule.customMOTD.extra.0": "使用'_'时将发送默认MOTD(从server.properties中读取)",
+  
+  "rule.defaultLoggers.name": "默认游戏监视器",
+  "rule.defaultLoggers.desc": "为所有新进入玩家设置部分监视器默认开启",
+  "rule.defaultLoggers.extra.0": "比如设置'tps,mobcaps'作为默认监视器",
+  
+  "rule.desertShrubs.name": "树苗在沙漠干枯",
+  "rule.desertShrubs.desc": "树苗将在附近无水源的沙漠群系枯萎",
+  
+  "rule.disableSpawnChunks.name": "允许出生点区块卸载",
+  "rule.disableSpawnChunks.desc": "允许出生点区块卸载",
+  
+  "rule.explosionNoBlockDamage.name": "防爆",
+  "rule.explosionNoBlockDamage.desc": "任何形式的爆炸都不会摧毁方块",
+  
+  "rule.extremeBehaviours.name": "随机极端情况",
+  "rule.extremeBehaviours.desc": "大幅度提升极端情况出现概率，仅用于测试",
+  "rule.extremeBehaviours.extra.0": "发射器和投掷器将使用最大随机刻",
+  
+  "rule.fastRedstoneDust.name": "红石粉卡顿优化",
+  "rule.fastRedstoneDust.desc": "红石粉卡顿优化 作者 Theosib",
+  
+  "rule.fillLimit.name": "fill/clone上限更改",
+  "rule.fillLimit.desc": "将fill/clone上限更改为自己需要的值",
+  "rule.fillLimit.extra.0": "你必须在1-20M之间取一个值",
+  
+  "rule.fillUpdates.name": "fill更新开关",
+  "rule.fillUpdates.desc": "fill/clone/setblock命令以及结构方块执行时是否产生更新",
+  "rule.fillUpdates.extra.0": "设置为true有更新 设置为false时为无更新",
+  
+  "rule.flatWorldStructureSpawning.name": "结构怪在超平坦世界生成",
+  "rule.flatWorldStructureSpawning.desc": "允许结构怪在超平坦世界生成",
+  
+  "rule.flippinCactus.name": "仙人掌扳手",
+  "rule.flippinCactus.desc": "允许使用仙人掌调整方块朝向，并且不会产生更新",
+  
+  "rule.hardcodeTNTangle.name": "硬编码TNT角度",
+  "rule.hardcodeTNTangle.desc": "把TNT水平随机角度设为固定值，可用于测试机器",
+  "rule.hardcodeTNTangle.extra.0": "默认为-1，必须在0与2Π之间，或者-1",
```

Comment:
The angle is in degrees, so spans from 0 to 360, not 2pi

Reply:
ok fixed.